pkgs/applications/networking/instant-messengers/webcord/default.nix

```diff
@@ -2,90 +2,92 @@
 , buildNpmPackage
 , fetchFromGitHub
 , copyDesktopItems
-, python3
-, pipewire
-, libpulseaudio
-, xdg-utils
 , electron_25
+, libpulseaudio
 , makeDesktopItem
 , nix-update-script
+, pipewire
+, python3
```

Comment:
just noticed this, has the convention for argument order changed from order of usage to alphabetical?

Reply:
I was told it's `lib` `stdenv`/ another builder then fetchers then all linux deps ASCII sorted then Darwin deps ASCII sorted

Reply:
Interesting, I always saw it as lib, stdenv, then arguments in order they're used first. I can't seem to find any docs on this but I'll look a bit harder.

Reply:
Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.

https://nixos.org/manual/nixpkgs/stable/#sec-syntax

Reply:
That section of the manual is now moved to CONTRIBUTING.md
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#code-convention
And no ordering is mentioned so idk what to think

Reply:
Ah, indeed, it has been removed in #247424, so there's no convention as of 23.11...

Reply:
Alright I'll stop recommending anything